tony-examples/tony-in-gcp/README.md

```diff
@@ -219,15 +219,93 @@ You can also track Job status from DataProc Jobs tab: Cloud Console -> Big Data
 
 ![img](https://i.imgur.com/3XgZI5Z.png)
 
+## Autoscaling
+
+

```

Comment:
worth clarifying this is for autoscaling the Hadoop cluster, not each individual TF or TonY job

Reply:
Ack